002_simulation_structure/002_simulation_structure.tex

```diff
@@ -1,14 +1,108 @@
 \chapter{Simulation Structure}\label{simulation_structure}
 
-\section{Simulation Environment}
+%%%%%%%%
+%%%%%%%% Simulation Environment
+%%%%%%%%
+\section{Simulation Environment}\label{simulation_environment}
 Goes over the backend design
 
-\section{Simulation Flow}
+%%%%%%%%
+%%%%%%%% Simulation Flow
+%%%%%%%%
+\section{Simulation Flow}\label{simulation_flow}
 The order in which events happen
 Need a nice diagram for this
 
-\section{Message Passing}
+%%%%%%%%
+%%%%%%%% Message Passing
+%%%%%%%%
+\section{Message Passing}\label{message_passing}
 Need a nice diagram for this
 
-\section{Health Decay System}
-Need a nice diagram for this
+%%%%%%%%
+%%%%%%%% Health Modeling
+%%%%%%%%
+\section{Health Modeling}\label{health_modeling}
+
+\ToDo{Add diagram with a possible scenario of several days in a row?}
+
+%
+%%%%%%%% Global Description
+%
+\subsection{Global Description}
+
+The health of the agents living in the tower is represented by Health Points (HP). Two mechanisms affect an agent's HP: how much food they eat, and their ``cost of living''. The cost of living represents how many calories a human needs to eat each day to stay healthy. These two mechanisms are implemented using the functions \lstinline$updateHP$ and \lstinline$hpDecay$, respectively. These two functions are described below.
+
+At the end of each day, agents are assigned an HP value based on how much food they have eaten and their cost of living. This HP value is an integer and has a maximum value of \lstinline$MaxHP$, and a minimum value of \lstinline$HPCritical$. As its name suggests, \lstinline$HPCritical$ is a critical HP value for the agents: they can only survive a certain number of days (\lstinline$MaxDayCritical$) at this level. When in the critical state, if agents can increase their HP by \lstinline$HPReqCToW$ (``HP Required to move from Critical To Weak''), then they move into the ``weak state'' (\Cref{fig:health_system}), and their HP takes the value of \lstinline$WeakLevel$. The amount that an agent's HP increases from eating is determined by the function \lstinline$updateHP()$.
+
+\begin{figure}[htb]
+    \centering
+    \includegraphics[width=0.3\linewidth]{002_simulation_structure/images/health_global.pdf}
```

Comment:
Is there a reason a zoomed in version of the diagram is shown instead of the full one? At the moment it seems like this doesn't need to be a diagram and it could just be a list of health levels.

Reply:
I am not to sure what you mean with "zoomed version". This diagram is the full one I think

Reply:
The image looks like its a graph with the other axis cropped out. That may just be my github not liking the uploaded image though

Reply:
![image](https://user-images.githubusercontent.com/80003671/148142275-19e43334-ca78-4bf2-9b3f-43f4dd65c542.png)

Reply:
It should be something like that. Is it what you also see? @Silvertrousers

Reply:
huh, I see this: 
![image](https://user-images.githubusercontent.com/56201218/148142441-34237fb5-53dd-475c-942a-b4713142a102.png)

Reply:
for the health_global.pdf file

Reply:
Fig 3.1 is indeed rather simple. We could have the special levels of the graph (WeakLevel, MaxHP,..) explained in the text only, but I think it is always helpful for the reader to have some diagrams. I think it is always good to have a bit too many diagrams too not enough, especially if we don't have a limited amount of space. 
What I would propose is to leave it there for now, and delete it if needed when the state of the report is more advanced

Reply:
alright, lets leave it in for now